qt/applications/workbench/workbench/plotting/globalfiguremanager.py

```diff
@@ -63,8 +70,14 @@ def destroy(cls, num):
         manager = cls.figs[num]
         manager.canvas.mpl_disconnect(manager._cidgcf)
 
-        if cls._active.num == num:
-            cls._active = None
+        # There must be a good reason for the following careful
+        # rebuilding of the activeQue; what is it?
```

Comment:
I think this comment is from `_pylab_helpers`, but if I understand this correctly it means we can do a sort by last active in the plot selector.

Reply:
I think you are correct and this will probably be welcome.